fdbserver/ptxn/TLogServer.actor.cpp

```diff
@@ -1566,6 +1553,16 @@ ACTOR Future<Void> tLog(
 						when(wait(waitForAll(tlogGroupRecoveries))) {}
 					}
 
+					// Disk errors need a chance to kill this actor.
+					wait(delay(0.000001));
+
+					for (auto& it : self->tlogGroups) {
+						Reference<TLogGroupData> tlogGroup = it.second;
+						tlogGroup->sharedActors.send(commitQueue(tlogGroup));
```

Comment:
Curious, why call `commitQueue` here, instead of one of the original two places?

Reply:
check [here](https://github.com/apple/foundationdb/blame/partitioned-txns/fdbserver/TLogServer.actor.cpp#L3359-L3373), non partitioned tlog code path would do
* first guarantee a good state of persistent storage
* then send commitQueue and updateStorageLoop actor


However, what we have now

Reply:
before this change, we did not wait for the tlog group persistent to be clear, instead we directly send actors. I am not sure whether it still works, but I think it is a good idea to keep the original order -- first wait for it to have a clear state(batch wait in this case), then send all actors. @jzhou77 let me know whether it makes sense

Reply:
This makes sense to me.